### PR TITLE
KG-51-game-design-without-ui

### PR DESCRIPTION
**Pull request by natalyTive**

### CODE REVIEW

File: src/data/animals_english.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
+const animals = [
+    {
+      id: 1,
+      name: 'ALLIGATOR',
+      shortName: 'A',
+      level: 'Hard',
```
[the remark this answers is not in the document]
Comment (marcelotivelabs) [reply src/data/animals_english.js:6]: That means that when other data needs to be used, a person must pre process the data and classify it, and that person can classify a long word at a low level or at another level.
But, the application can classify the data according to the length of the words for example, and thus the levels would not necessarily be in the data.

**Review by marcelotivelabs — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import hangedManUtils from "./src/util/hangedManUtils.js";
+
+let hangedMan;
+export function initHangedMan(category, level) {
```
Comment (marcelotivelabs) [inline index.js:4]: Please add some validations for the function's incoming parameters.

**Review by marcelotivelabs — CHANGES_REQUESTED** [review]

File: src/util/arrayUtils.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+            default:
+                break;
+        }
+
+        return itemsList;
+    },
+    fillAnswerWithUnderscoreLetters(word) {
+        let answer = [];
+        const myArray = word.split('');
+        myArray.forEach(e => {
```
Comment (vbrianh) [inline src/util/arrayUtils.js:32]: remove unnecessary variable e, example use _

File: src/util/arrayUtils.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      indexes.forEach(index => {
+        answer[index] = letter;
+      });
+  
+      return answer;
+    },
+    getItemsListByType (category, level) {
+        let itemsList = [];
+        switch (category) {
+            case 'Animals':
```
Comment (vbrianh) [inline src/util/arrayUtils.js:17]: move to constants, example

const categories = {
    animals: 'Animals',
    numbers: 'Numbers',
};

case categories.animals

File: test/util/hangedManUtils.test.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import hangedManUtils from "../../src/util/hangedManUtils";
```
Comment (vbrianh) [inline test/util/hangedManUtils.test.js:1]: i have the next error 
Screenshot 2023-01-06 at 05 27 57

**Review by vbrianh — CHANGES_REQUESTED** [review]

**Review by vbrianh — APPROVED** [review]
LGTM!

**Review by tiveor — APPROVED** [review]
LGTM

**Pull request merged**